.github/workflows/dockerimage.yml

```diff
@@ -38,6 +38,10 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
+      - name: Authenticate Github Packages
+        run: |
+          echo "<settings><servers><server><id>github</id><username>${{ github.actor }}</username><password>${{ secrets.GITHUB_TOKEN }}</password></server></servers></settings>" > settings.xml
```

Comment:
It feels like puting token into a `settings.xml` file inside our docker container is not an ideal solution